tests/Main.hs

```diff
@@ -411,10 +456,15 @@ main = do
           proc
             "consul"
             [ "agent", "-dev"
+            , "-node", (unpack localhost) -- hardcode node name as "localhost"
```

Comment:
A comment _why_ this is needed/useful would be helpful here.

Reply:
Ah yes, that's not obvious. For notes and easy copying when I make the update: When we create a session, we need to reference a Node that has been registered in Consul's node catalog. By telling the agent to use `localhost`, after the agent boots, we can expect that the agent has registered a node for itself and that the node's name is `localhost`, so that when we create a session, we can simply reference that existing/registered node from the agent instead of having to make up and register a Node for the test.

Reply:
Comment added.